site/docs/intro/under-the-hood.md

```diff
@@ -2,5 +2,55 @@
 sidebar_position: 4
 title: Under the hood
 ---
+## Provisioning preview environments
 
-# Under the hood
+When provisioning a new environment using the `preevy up` command the following steps are taken:
+- `preevy` profile store is read for default configuration and relevant keys.
```

Comment:
Some of these points starts with the active voice `preevy does...` or `the CLI` and some with the passive `something is done`.
I suggest using the active voice with an implicit actor:

- Calculates an environment ID
- Creates a machine
- Installs docker...

Reply:
done